README.md

```diff
@@ -172,7 +172,11 @@ steps:
     with:
       go-version: '1.17'
       check-latest: true
-      cache-dependency-path: subdir/go.sum
+      cache-dependency-path: |
+             subdir/go.sum 
+             tools/go.sum   
```

Comment:
this syntax is pretty unclear. Is this parameter accepting space-separated strings? Or an array? Something like this would be more idiomatic yaml:
```
      cache-dependency-path:
        - subdir/go.sum 
        - tools/go.sum
```

Reply:
@yurishkuro This format inherited from [cache](https://github.com/actions/cache) action as I understood: https://github.com/actions/cache#using-a-combination-of-restore-and-save-actions

So I think it make sense to open an issue there.